core/src/main/kotlin/model/additionalExtras.kt

```diff
@@ -61,7 +61,7 @@ data class Annotations(
         }.toMap()
 
     enum class AnnotationScope {
-        DIRECT, FILE
+        DIRECT, FILE, GETTER, SETTER
```

Comment:
why do you need those?

Reply:
I use these scopes to render in https://github.com/Kotlin/dokka/pull/2049/files#diff-8cd255845347168a7599ddf3b534ad987e31b9d92fe4e16303279cd3b6172ca9R74
Or I can introduce an extra parameter of that function to define the annotation target?